hash_practice/exercises.py

```diff
@@ -1,19 +1,43 @@
 
+from ntpath import join
+from sqlalchemy import true

```

Comment:
Looks like these `import` statements never got used, so you can remove them to keep the code clean as a good style practice.